src/components/mediaManager/AposMediaManagerDisplay.vue

```diff
@@ -59,13 +64,33 @@ export default {
     CloudUpload
   },
   mixins: [ AposHelpers ],
+  // Custom model to handle the v-model connection on the parent.
+  model: {
+    prop: 'checked',
+    event: 'change'
+  },
   props: {
+    checked: {
+      type: [Array, Boolean],
+      default: false
+    },
     media: {
       type: Array,
       default() {
         return [];
       }
     }
+  },
+  computed: {
```

Comment:
AposMediaManager has this extra layer, so I'm doing the same things as with AposCheckbox to keep `v-model` in sync.